content/docs/rfcs/22/README.md

```diff
@@ -0,0 +1,46 @@
+---
+slug: 22
+title: 22/CHAT2
```

Comment:
Is chat2 a good name? I gave it this name because of naming convention in nim-waku with 1 being for Waku v1 and 2 being for Waku v2. Maybe call it "TOY-CHAT"?